Rules/AvoidInvokingEmptyMembers.cs

```diff
@@ -20,14 +20,15 @@
 
 namespace Microsoft.Windows.PowerShell.ScriptAnalyzer.BuiltinRules
 {
+    // Rule name is inconsistent.
     /// <summary>
-    /// AvoidAlias: Check if cmdlet alias is used.
-    /// </summary>
+    /// AvoidInvokingEmptyMembers: Analyzes the script to check if any non-constant members have been invoked.
```

Comment:
Kapil. Please verify that the comment matches the rule in all the other rules and open one issue to track the fix.